src/Evolution/Systems/Ccz4/Christoffel.cpp

```diff
@@ -0,0 +1,65 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <cstddef>
+
+#include "Evolution/Systems/Ccz4/Christoffel.hpp"
```

Comment:
this should be before the system includes. See, eg https://spectre-code.org/code_review_guide.html (search for "Header order")